### PR TITLE
Add Tribut-Run-Buffs

### PR DESCRIPTION
**Pull request by Nolin7777**

### CODE REVIEW

File: Updates/1567_tribut_buffs.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	VALUES
+		('60000', 'You are da King of the Gordok! You''re A-number-one!', NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL),
```
Comment (Tobschinski) [inline Updates/1567_tribut_buffs.sql]: already exists, id 6907

File: Updates/1567_tribut_buffs.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		('60001', 'You da man now, dog!', NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL),
```
Comment (Tobschinski) [inline Updates/1567_tribut_buffs.sql]: already exists, id 6903

File: Updates/1567_tribut_buffs.sql [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+		('60002', 'You got me good boss! How you get by Slip''kik?? I smart though. I watch next time! Har har!$B$B Nobody get by Slip''kik no more. Slip''kik promise!', NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, '0', '0', NULL, NULL, NULL, NULL, NULL, NULL);
```
Comment (Tobschinski) [inline Updates/1567_tribut_buffs.sql]: already exists, id 6905

**Comment by cala** [comment]
Thank you very much. I'll test this next week.

**Comment by Tobschinski** [comment]
```
ServerToClient: SMSG_GOSSIP_MESSAGE (0x1746)
GossipGUID: Full: - Creature/0 R-/S- Map: 429 Entry: 14321 Low: -
GossipID: 5731
FriendshipFactionID: 0
TextID: 6903
GossipOptions: 0
GossipText: 0
```

```
ServerToClient: SMSG_GOSSIP_MESSAGE (0x1746)
GossipGUID: Full: - Creature/0 R-/S- Map: 429 Entry: 14326 Low: -
GossipID: 5735
FriendshipFactionID: 0
TextID: 6907
GossipOptions: 0
GossipText: 0
```

I don't have any information about the 3rd Guard, however.

**Comment by Nolin7777** [comment]
I am sorry, I actually wrote this for MaNGOS Zero.
I will compile the CMaNGOS Core and try to make this work for you guys.

**Comment by cala** [comment]
Merged in https://github.com/cmangos/classic-db/commit/4e95e5797869f24b576298e1e64d9f3cdcf2e8d2
Thanks @Tobschinski for the correct IDs.
I made a few adjustements on the original PR:
- used correct gossip IDs (based on @Tobschinski research)
- fixed the flags of the DBScripts (players were casting the buffs onto the guards in the previous state)
- fixed gossip menu option logic

Thanks @Nolin7777

**Pull request closed without merging**